Classes/Views/DiffString.swift

```diff
@@ -7,6 +7,35 @@
 //
 
 import UIKit
+import Highlightr
+
+func CreateColorCodedString(code: String, includeDiff: Bool=false) -> NSAttributedString {
+    guard let highlightr = Highlightr() else { return NSAttributedString(string: code) }
+
+    if includeDiff {
+        highlightr.setTheme(to: "github")
+        let colorCodedString = highlightr.highlight(code, as: "swift") ?? NSAttributedString(string: code)
```

Comment:
Don't think we can/should assume the language, could be anything! Objective-C 😷

Reply:
You can automatically detect the language but then it didn't looked that good. See:
![Image](https://i.imgur.com/cc04t2g.png)